server/src/app.ts

```diff
@@ -81,6 +82,18 @@ export const createApp = (): express.Application => {
     const hashed = qs.unescape(req.params.hashed);
 
     try {
+      const user = await prisma.user.findOne({
+        where: {
+          email,
+        },
+      });
+      const alreadyVerified = user && user.verified;
+
+      if (alreadyVerified) {
+        res.send('Already verified email');
```

Comment:
What about `res.sendStatus(404)`?

Reply:
Other users & user with the `url` might not need to know if current email is already verified.

Reply:
I changed this code to `res.sendStatus(404)`